build/cmake_deps.toml

```diff
@@ -386,10 +386,7 @@ buck_targets = [
   "//extension/llm/custom_ops:custom_ops",
 ]
 filters = [
-  # Second clause is to pick up fht_neon.c/fht_avx.c from FFHT. TODO:
-  # remove filters and patch extract_sources.py's Buck query to fetch
-  # srcs; presumably filters is here to remove .h files.
-  "(.cpp$)|(fht.*\\.c$)",
```

Comment:
My comment is irrelevant since you are removing it, but this second regex seemed wrong?

Reply:
Looks right to me. Dot is a regex metacharacter so we need to escape it with a backslash. Backslash is a string metacharacter so we need to escape it with a second backslash. Or is there a different problem I missed entirely?